core-api/ormconfig.json

```diff
@@ -1,15 +1,16 @@
 {
     "type": "postgres",
-    "host": "core-db",
+    "host": "ec2-54-235-167-210.compute-1.amazonaws.com",

```

Comment:
It's import to put all that on a .env file, exchange this parameter for something like "process.env.CORE_DB_HOST || core-db". This way this changes will not affect the local execution.

Reply:
In order to do that you should change the extension of this file to `.js`, because you can't do that in plain json. Check it out: https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md#using-ormconfigjs

Reply:
Done, using environment variables directly from heroku